test/inductor/test_torchinductor.py

```diff
@@ -7221,6 +7221,20 @@ def fn_channels_last(x):
             [torch.randn(8, 384, 20, 20).to(memory_format=torch.channels_last)],
         )
 
+    def test_exact_stride(self):
+        full = torch.randn((16, 16), device=self.device)
+        view = torch.as_strided(full, (16, 8), full.stride())
+
+        def fn(x):
+            result = x + x
+            result_strided = torch.empty_strided(
+                x.size(), x.stride(), device=self.device
+            )
+            result_strided[:] = result
+            return result_strided
+
+        self.common(fn, [view])
```

Comment:
Should we assert that the output stride is `(16,1)`?

Reply:
yes. will fix it in next commit.